source.js

```diff
@@ -22,6 +22,11 @@
 				};
 
 				this.source.$textarea = $('<textarea />');
+				// This line here is a fix so that we can use ng-model binding for the code view
+				// for the redactor class. It basically grabs the original textarea field and binds
+				// it to the text area of the code view in the redactor so that the ng-model bind will
+				// be bound to the code view as well as the rich content editor view.
+				this.source.$textarea[0] = this.core.box()[0].childNodes[3];
```

Comment:
is there a way to find the node by name/id/tag or something else?, finding element this way (by indexing arrays) is kind of error prone.

Reply:
this is a good point. It's a good practice to add a class name for it that's only for javascript, e.g. `js-my-class`